packages/browser/src/loader.js

```diff
@@ -128,13 +128,14 @@
       // Because we installed the SDK, at this point we have an access to TraceKit's handler,
       // which can take care of browser differences (eg. missing exception argument in onerror)
       var tracekitErrorHandler = _window[_onerror];
+      var tracekitUnhandledRejectionHandler = _window[_onunhandledrejection];
 
       // And now capture all previously caught exceptions
       for (var i = 0; i < data.length; i++) {
-        if (data[i].e) {
+        if (data[i].e && tracekitErrorHandler) {
           tracekitErrorHandler.apply(_window, data[i].e);
-        } else if (data[i].p) {
-          SDK.captureException(data[i].p);
+        } else if (data[i].p && tracekitUnhandledRejectionHandler) {
+          tracekitUnhandledRejectionHandler.apply(_window, [data[i].p]);
```

Comment:
Why do we I need to pass `[data[i].p]` as an array here?
If I don't do this it doesn't work. This is my only question 🤔

Reply:
It's because `data[i].e`, is an array of arguments retrieved from `onerror` handler, where `data[i].p` is a `rejection.reason` attribute itself. `apply` always accepts an array as the second argument. It can be changed to `tracekitUnhandledRejectionHandler.call(_window, data[i].p);`

Reply:
We can modify https://github.com/getsentry/sentry-javascript/blob/a6176f513fc4f4173573f9844d05a0867cba08d0/packages/browser/src/loader.js#L183-L200 to store all arguments as well and apply them in the same manner if we want to unify this